packages/winpanda/extra/src/winpanda/tests/unit/test_common_utils.py

```diff
@@ -0,0 +1,163 @@
+import pytest
+import mock
+
+from common import utils
+from common.exceptions import ExternalCommandError
+from subprocess import SubprocessError, CalledProcessError
+
+
+@mock.patch('common.utils.SmartDL')
+def test_download(*args):
+    """
+    Download fake content and save it into file.
+    """
+    path = utils.download('', 'path')
+    assert isinstance(path, str)
+
```

Comment:
More reasonable would be to check that a returned string represents an absolute path.

Reply:
replaced by 'endswith' because of absolute path.

Reply:
redesigned